docs/add-ins/get-or-set-the-location-of-an-appointment.md

```diff
@@ -1,30 +1,111 @@
 ---
 title: Get or set the location of an appointment in an add-in
 description: Learn how to get or set the location of an appointment in an Outlook add-in.
-ms.date: 09/23/2019
+ms.date: 10/31/2019
 localization_priority: Normal
 ---
 
 # Get or set the location when composing an appointment in Outlook
 
 The JavaScript API for Office provides properties and methods to manage the location of an appointment that the user is composing. Currently, there are two properties that provide an appointment's location:
 
-- [item.location](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/office.context.mailbox.item#location-stringlocation): Basic API that allows you to get and set the location.
-- [item.enhancedLocation](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/office.context.mailbox.item#enhancedlocation-enhancedlocation) ([preview](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview)): Enhanced API that allows you to get and set the location, and includes specifying the [location type](/javascript/api/outlook/office.mailboxenums.locationtype). The type is `LocationType.Custom` if you set the location using `item.location`.
+- [item.location](/office/dev/add-ins/reference/objectmodel/requirement-set-1.1/office.context.mailbox.item#location-stringlocation): Basic API that allows you to get and set the location.
```

Comment:
Should this be 1.8, not 1.1?

Reply:
It was introduced in 1.0. I've been setting the reqt. set to when the API was introduced so that I wouldn't update them subsequently. Alternatives are to update to most recent released version periodically or always use preview reqt. set. Any preference?

Reply:
My feeling is that we should we should send people to the latest. Preview will always be the latest, so it's both complete and maintainable.

Reply:
I'll scrub in a subsequent change since there's some other content that will likely need to be updated too.